medical_designs.dm

```diff
@@ -0,0 +1,401 @@
+/////////////////////////////////////////
+////////////Medical Tools////////////////
```

Comment:
You made a new medical_designs.dm for some reason. 

[See?](http://i.imgur.com/mq6GnQD.png)